### PR TITLE
Replaced addValue to setValue for file upload

### PR DESCRIPTION
**Pull request by Ananthreddy3004**
const remoteFilePath = await browser.uploadFile(filePath);
await elem.addValue(remoteFilePath);  

addValue method will not clear the textbox so in first iteration it will upload one file **test.txt** and in second iteration it will upload new file **test1.txt** along with **test.txt** because this file already exist in textbox so we can see duplicate of file upload.

if we use the below this things in place of addValue it will work as expected
await elem.setValue(remoteFilePath);

### CODE REVIEW

**Review by heraldantony — APPROVED** [review]
looks good

File: src/reuse/modules/util/file.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export class File {
         const filePath = path.resolve(file);
         vl.log(`Uploading file with a path ${filePath}`);
         const remoteFilePath = await browser.uploadFile(filePath);
-        await elem.addValue(remoteFilePath);
+        await elem.setValue(remoteFilePath);
```
Comment (DIFSRIP) [inline src/reuse/modules/util/file.ts:47]: @heraldantony mentioned, that setValue is only possible for displayed elements. Are we sure it is working with that change for hidden inputs?

Comment (Ananthreddy3004) [reply src/reuse/modules/util/file.ts:47]: @DIFSRIP for hidden element explicitly they have handle it externally to make visible samething we followed in customerE2E.if we want me to handle it at util level then I will do it.

Comment (DIFSRIP) [reply src/reuse/modules/util/file.ts:47]: are there many such cases where it is not visible, do we have concrete samples?

Comment (DIFSRIP) [reply src/reuse/modules/util/file.ts:47]: We can improve it in a separate PR if so, need to release the fix soon, so I will merge it like this for now.

**Pull request merged**